static/app/components/searchSyntax/parser.tsx

```diff
@@ -320,6 +320,11 @@ type FilterMap = {
      * The value of the filter
      */
     value: KVConverter<FilterTypeConfig[F]['validValues'][number]>;
+    /**
+     * A warning message associated with this filter
+     * The FE will render it as a tooltip
```

Comment:
```suggestion
```

Let's leave out specifically what is done here, since that's a few abstraction levels up from what the searchSyntax parser's concerns